index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/styles.css" />
+  <title>My Portifolio</title>
+</head>
+
+<body>
+  <header>
+    <nav class="flexr container">
+      <a href="#" class="link">
+        <img src="https://arsenethierry-dev.vercel.app/images/logo-light.svg" width="90px" height="60px"
+          alt="github icon" />
+      </a>
+      <img src="/images/menu button.png" width="18px" height="12px" id="menu" alt="menu icon" />
```

Comment:
- [OPTIONAL ]I can see that you made the changes but your design didn't reflect the Figma design, Your logo looks different from the original design, as you can see in the image below, kindly make sure that the fonts, and colors are reflecting the Figma design, please fix this :point_down: 

Your Design 
![Screenshot from 2023-04-29 03-23-00](https://user-images.githubusercontent.com/88894525/235277223-6a5c66a1-5e0d-42ea-af25-c5008330bd0d.png)

Figma Design 
![Screenshot from 2023-04-29 03-24-20](https://user-images.githubusercontent.com/88894525/235277278-42d36914-b099-49b9-ac4b-c756626b94c8.png)